packages/apps/tools/src/components/Common/Layout/index.tsx

```diff
@@ -31,13 +50,91 @@ export const Layout: FC<IProps> = ({ children }: IProps) => {
     },
   ];
 
+  const getNetworks = (): Network[] => {
```

Comment:
This logic is quite specific and belongs to the connect wallet functionality, right? If so; wouldn't it maybe be better to extract this logic to a better location?

Reply:
not really, here we just want to get accounts that belong to a chain, wallet connect gives you all of the accounts with network as its prefix, so we want to visually separate them not logically